### ISSUE TITLE
CALICO_IPV4POOL_CIDR will accept pools which calicoctl will not

### ISSUE THREAD

**Issue by lwr20** [projectcalico/calicoctl#1537]
I run this, expecting it to fail because I've asked for a pool which is too small (10.0.0.0/32):
`$ docker run --net=host --privileged --name=calico-node -d --restart=always -e IP_AUTODETECT_METHOD=first-found -e CALICO_LIBNETWORK_CREATE_PROFILES=true -e CALICO_LIBNETWORK_LABEL_ENDPOINTS=false -e CALICO_LIBNETWORK_IFPREFIX=cali -e NODENAME=drogon -e CALICO_NETWORKING_BACKEND=bird -e CALICO_IPV4POOL_CIDR=10.0.0.0/32 -e CALICO_NAT_ENABLED=false -e CALICO_LIBNETWORK_ENABLED=true -e IP6_AUTODETECT_METHOD=first-found -e ETCD_AUTHORITY=127.0.0.1:2379 -e ETCD_SCHEME=http -e ETCD_ENDPOINTS= -v /var/log/calico:/var/log/calico -v /var/run/calico:/var/run/calico -v /lib/modules:/lib/modules -v /run/docker/plugins:/run/docker/plugins -v /var/run/docker.sock:/var/run/docker.sock quay.io/calico/node:v1.1.0-rc5
`

But it succeeds:
```
$ ./calicoctl get ippool -o yaml
- apiVersion: v1
  kind: ipPool
  metadata:
    cidr: 10.0.0.0/32
  spec:
    nat-outgoing: true
- apiVersion: v1
  kind: ipPool
  metadata:
    cidr: fd80:24e2:f998:72d6::/64
  spec:
    nat-outgoing: true
```

If I try to create that pool via calicoctl, it is rejected (as expected):
```
$ ./calicoctl get ippool -o yaml > pool
$ ./calicoctl apply -f pool
Failed to execute command: error with field CIDR = '10.0.0.0/32' (IP pool is too small)
```

**Comment by lwr20** [comment]
Similarly for `CALICO_IPV6POOL_CIDR` accepting `fd00::/123`

**Comment by tmjd** [comment]
Ready for retest with #1542 merged

**Comment by lwr20** [comment]
Fix looks good in calico/node:v1.1.0-rc5-14-g1f5381f3
